OpenScienceJournal/whistlepunk_library/src/main/java/com/google/android/apps/forscience/whistlepunk/project/experiment/ExperimentDetailsFragment.java

```diff
@@ -596,7 +597,14 @@ public void onPrepareOptionsMenu(Menu menu) {
       menu.findItem(R.id.action_download_experiment).setEnabled(!isRecording());
       setHomeButtonState(isRecording());
     } else {
-      setHomeButtonState(!activity.isToolFragmentVisible() && isRecording());
+      if (activity.isSensorFragmentVisible() && isRecording()) {
+        final Drawable v =
```

Comment:
In the single greatest nitpick of my career, why is this "v"?

Reply:
Because the drawable resolves to be a down arrow that actually looks like a v :)